package.json

```diff
@@ -25,17 +28,20 @@
     "index.js"
   ],
   "license": "MIT",
+  "dependencies": {
+    "ember-cli-babel": "^5.1.5"
+  },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.3",
-    "coveralls": "^2.11.13",
-    "ember-cli": "2.9.1",
+    "coveralls": "^2.11.15",
+    "ember-cli": "^2.10.0",
```

Comment:
Hmm, interesting. I do the dependency updates for `ember-ajax` through Yarn, and that is what it changed it to by default.